Github/legos/github_create_issue/github_create_issue.py

```diff
@@ -0,0 +1,85 @@
+##
+##  Copyright (c) 2023 unSkript, Inc
+##  All rights reserved.
+##
+
+import pprint
+from typing import Optional, List
+from pydantic import BaseModel, Field
+from github import GithubException
+
+
+class InputSchema(BaseModel):
+    owner: str = Field(
+        description='Username of the GitHub user. Eg: "johnwick"', 
+        title='Owner'
+    )
+    repository: str = Field(
+        description='Name of the GitHub repository. Eg: "Awesome-CloudOps-Automation"',
+        title='Repository',
+    )
+    title: str = Field(
+        description='Title if the Github Issue', 
+        title='Title of the Issue'
+    )
+    description: str = Field(
+        description='Description of the Github Issue', 
+        title='Title of the Issue'
+    )
+    assignee: str = Field(
+        description='Username of the Github User to assign this issue to ', 
+        title='Username of the Assignee'
+    )
+
+
+
+def github_create_issue_printer(output):
+    if output is None:
+        return
+    pprint.pprint(output)
+
+
+def github_create_issue(handle, owner:str, repository:str, title:str, description:str, assignee: str) -> List:
```

Comment:
Why is return a List, are we creating list of issue?

Reply:
It returns the details of the newly created issue like issue no, title and assignee, which is a list of dicts

Reply:
Changed it @amit-chandak-unskript